english/BayesTheorem/BayesTheorem.md

```diff
@@ -2,4 +2,6 @@ Bayes' Theorem: P(A|B) = P(B|A)P(A)/P(B)
 
 [Daddy, but where do priors come from](priors/priors.md)?
 
-[Daddy, can I have a glass of water](../coffee/drink-water/drink.md)?
\ No newline at end of file
+[Daddy, can I have a glass of water](../coffee/drink-water/drink.md)?
+
+[Daddy, are there priors in my chocolate](../../chocolateOrSleep/chocolateOrSleep.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.